student.rb

```diff
@@ -0,0 +1,16 @@
+require_relative 'person'
+
+class Student < Person
+  def initialize(classroom, age, parent_permission, name = 'Unknown')
+    @classroom = classroom
+    super(
+      @age = age,
+      @parent_permission = parent_permission,
+      @name = name
+  )
```

Comment:
- As the previous reviewer mentioned! Please refactor this block of code exactly as you did with the super method in the Teacher class constructor. 🙏